.github/workflows/ci.yml

```diff
@@ -210,7 +210,8 @@ jobs:
           dfly-executable: dragonfly
           run-only-on-ubuntu-latest: true
           build-folder-name: build
-          filter: ${{ matrix.build-type == 'Release' && '(not slow) and (not dbg_only)' || '(not slow) and (not opt_only)' }}
+          # Non-release build will not run tests marked as slow or opt_only
+          filter: ${{ matrix.build-type == 'Release' && 'not empty' || '(not slow) and (not opt_only)' }}
```

Comment:
not empty is needed because we can not use empty filter

Reply:
of course we dont have any empty marker in our code

Reply:
I would consider leaving a comment for this -- we will forget it in one week from now :rofl: